stories/busy-indicator/busy-indicator.stories.js

```diff
@@ -68,6 +69,37 @@ export const contrastMode = () => `<div style="display:flex;justify-content:cent
 contrastMode.parameters = {
     docs: {
         iframeHeight: 200,
-        storyDescription: 'The busy indicator also comes in contrast mode and displays white dots against a dark background. To apply contrast mode, add <code>contrast</code> into the element i.e. <code>fd-busy-indicator--m contrast</code>.'
+        storyDescription:
+            'The busy indicator also comes in contrast mode and displays white dots against a dark background. To apply contrast mode, add <code>contrast</code> into the element i.e. <code>fd-busy-indicator--m contrast</code>.'
+    }
+};
+export const LabelledIndicator = () => `<div style="text-align: center">
+    <div class="fd-busy-indicator fd-busy-indicator--l" aria-hidden="false" aria-label="Loading">
+        <div class="fd-busy-indicator--circle-0"></div>
+        <div class="fd-busy-indicator--circle-1"></div>
+        <div class="fd-busy-indicator--circle-2"></div>
+		<span class="fd-busy-indicator--label">loading data...</span>
+    </div>`;
+
+LabelledIndicator.parameters = {
+    docs: {
+        iframeHeight: 200,
+        storyDescription:
+            'The standard busy indicator animates a sequence of cascading dots expanding and shrinking in a loop. The component comes with a label/message to be displayed when the data is being loaded. <code>fd-busy-indicator--label</code> is used to display the message.'
+    }
+};
+export const ExtendedIndicator = () => `<div style="text-align: center">
+    <div class="fd-busy-indicator--l fd-busy-indicator--extended-indicator fd-padding" aria-hidden="false" aria-label="Loading">
```

Comment:
You can't have this: `<div class="fd-busy-indicator--l fd-busy-indicator--extended-indicator fd-padding">`
You need a base class to modify it:
<div class="fd-busy-indicator fd-busy-indicator--l fd-busy-indicator--extended-indicator fd-padding" aria-hidden="false" aria-label="Loading">

Reply:
@InnaAtanasova  on checking the previous example found that it does not require any base class, so a new modifier class has been added. 

![image](https://user-images.githubusercontent.com/53534379/118518068-66384900-b755-11eb-855b-df81115b40ed.png)

Reply:
changed fd-busy-indicator--extended-indicator to fd-busy-indicator__extended

Reply:
It should be `class="fd-busy-indicator d-busy-indicator--l fd-busy-indicator--extended"`

`fd-busy-indicator__extended` means a separate element, in your case it's a modifier of `fd-busy-indicator`

Reply:
What is the purpose of extended busy indicator?

Reply:
busyindicator has a box-shadow property add to it giving it a view as shown in the screenshot.

![image](https://user-images.githubusercontent.com/53534379/118796760-88e37280-b8b9-11eb-9c07-6f233dc84442.png)